src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/SetAzureSqlFailoverGroup.cs

```diff
@@ -44,36 +54,38 @@ public class SetAzureSqlFailoverGroup : AzureSqlFailoverGroupCmdletBase
         /// Gets or sets the failover policy without data loss for the Sql Azure Failover Group.
         /// </summary>
         [Parameter(Mandatory = false,
-            HelpMessage = "The failover policy without data loss for the failover group.")]
+            HelpMessage = "The failover policy of the Azure SQL Database Failover Group.")]
         [ValidateNotNullOrEmpty]
         public FailoverPolicy FailoverPolicy { get; set; }
 
         /// <summary>
         /// Gets or sets the grace period with data loss for the Sql Azure Failover Group.
         /// </summary>
         [Parameter(Mandatory = false,
-            HelpMessage = "The grace period for failover with data loss of the failover group. This property defines how big of the window we tolerate for data loss during failover operation")]
+            HelpMessage = "The grace period during outage before automatic failover with data loss of the Failover Group is triggered.")]
+        [ValidateNotNullOrEmpty]
+        [ValidateRange(0, int.MaxValue)]
+        public int GracePeriodWithDataLossHours { get; set; }
+
+        /// <summary>
+        /// Gets or sets the grace period with data loss for the Sql Azure Failover Group.
+        /// </summary>
+        [Parameter(Mandatory = false, DontShow = true,
+            HelpMessage = "The grace period during outage before automatic failover with data loss of the Failover Group is triggered.")]
         [ValidateNotNullOrEmpty]
+        [ValidateRange(0, int.MaxValue)]
         [Obsolete("This parameter will be deprecated in the next release.")]
         public int GracePeriodWithDataLossHour { get; set; }
```

Comment:
This wasn't removed from the other cmdlet -- the github diff is obscuring the fact that it existed previously in both.